src/bluetooth/SensorManager.js

```diff
@@ -0,0 +1,60 @@
+// Default sensor generator.
+const createSensor = ({ id, macAddress, name, location, batteryLevel, isActive, logInterval }) => ({
+  id,
+  macAddress,
+  name,
+  location,
+  batteryLevel,
+  isActive,
+  logInterval,
+});
+
+class SensorManager {
+  constructor(dbService, utils) {
```

Comment:
```
import {generateUUID} from 'react-native'

... dbServiceutils, utils = { createUuid: () => generateUUID() })
...
```

At the moment sensors have no ID when created and realm throws an error for me

Reply:
not getting that. do you have the vaccineModule enabled?

Reply:
Ohh!! Doh!